pom/playwrightPage.js

```diff
@@ -0,0 +1,75 @@
+export class playwrightPage {
+  constructor(page) {
+    this.page = page;
+
+    /* todo components */
+    this.todoInsertField = page.getByPlaceholder('What needs to be done?');
+    this.todoItemList = page.getByTestId('todo-title')
+    this.removeTodoItem = page.getByRole('button', { name: 'Delete' });
+    this.deleteButton = page.getByRole('button', { name: 'Delete' });
+    this.nrOfItemsInTodoList = page.locator('span[data-testid="todo-count"]').nth(0).locator('strong');
+    this.checkBoxItem = page.locator('.toggle').first();
+    this.activeButton = page.getByRole('link', { name: 'Active' });
+    this.completedButton = page.getByRole('link', { name: 'Completed' });
+    this.clearCompletedButton = page.getByRole('button', { name: 'Clear completed' });
+    this.todoEditItem = page.getByTestId('todo-item');
+    this.taskTextBox = page.getByRole('textbox', { name: 'Edit' }); //this is the locator
+
+    /* footer links */
+    this.createdByLink = page.getByRole('link', { name: 'Remo H. Jansen' });
+    this.partOfLink = page.getByRole('link', { name: 'TodoMVC', exact: true });
+
+    /* header links */
+    this.headerLink = page.getByRole('link', { name: 'real TodoMVC app.' });
+
+  }
+
+  async accessPage() {
+    await this.page.goto('https://demo.playwright.dev/todomvc');
+    await this.page.context().clearCookies();
```

Comment:
why do you need to clear cookies ?

Reply:
I don't, Vio explained me why you are using it in your framework. Thank you!